src/GitLink/Interfaces/ISolutionParser.cs

```diff
@@ -0,0 +1,11 @@
+namespace GitLink.Interfaces
```

Comment:
So I also exclude the folder with Resharper shared settings? I mean the [Namespace provider](http://stackoverflow.com/questions/6821579/where-does-resharper-store-the-namespace-provider-settings) option